kotsadm/pkg/handlers/metadata.go

```diff
@@ -22,6 +22,7 @@ type MetadataResponse struct {
 	IsKurlEnabled bool   `json:"isKurlEnabled"`
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
this is needed for branding/some cluster flags, before user is logged in